### PR TITLE
[9.0][ADD] DPD carrier

### PR DESCRIPTION
**Pull request by hparfr**
DPD is a carrier company: https://www.dpd.com/home/about_us

This PR adds 2 modules : 
delivery_roulier
delivery_roulier_dpd

Roulier is a python lib. It's an API wrapper around multiple carriers.
delivery_roulier is the core module.
delivery_roulier_dpd  adds specifics fields for DPD.

For the moment, only DPD France is supported.
Other modules for Laposte and Geodis will come soon.

### CODE REVIEW

**Comment by bealdav** [comment]
ping @yvaucher

File: delivery_roulier/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+#  @author Raphael Reverdy <raphael.reverdy@akretion.com>
+#          David BEAL <david.beal@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Delivery Carrier Roulier',
+    'version': '9.0.0.0.0',
```
Comment (EBII) [inline delivery_roulier/__openerp__.py]: not started ?

**Review by EBII — COMMENTED** [review]
Nothing else  version number

File: delivery_roulier/README.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`Roulier <https://pypi.python.org/pypi/roulier>`_ is a python library which implements carriers API.
+This modules contains the core functions for this implementation.
+
+You should install one of the specific modules : 
+
+- delivery_roulier_laposte
+- delivery_roulier_dpd
+- delivery_roulier_geodis
+- more to come
```
Comment (yvaucher) [inline delivery_roulier/README.rst]: See https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

For readme format

**Comment by bealdav** [comment]
@hpar, please close/reopen PR to make re run runbot

**Comment by bealdav** [comment]
Runbot result

+=======================================
|  Tests summary:
|---------------------------------------
| test_server.py              [1;32mSuccess[0;m
+=======================================

**Review by bealdav — APPROVED** [review]

**Review by yvaucher — CHANGES_REQUESTED** [review]
LGTM appart readme format

File: delivery_roulier/README.rst (hunk 2 above)
Comment (bealdav) [reply delivery_roulier/README.rst]: static README template is the evil https://github.com/OCA/maintainer-tools/issues/238

**Comment by bealdav** [comment]
ping @hparfr could you update readme please, thanks

**Comment by hparfr** [comment]
Readme updated and some fixes added

**Comment by bealdav** [comment]
2018-03-19 17:02:08,376 187 INFO openerp_test openerp.modules.loading: loading delivery_roulier_dpd/data/delivery.xml
2018-03-19 17:02:08,580 187 WARNING openerp_test openerp.models: delivery.carrier.create() includes unknown fields: use_detailed_pricelist
2018-03-19 17:02:08,868 187 WARNING openerp_test openerp.models: delivery.carrier.create() includes unknown fields: use_detailed_pricelist
2018-03-19 17:02:09,129 187 WARNING openerp_test openerp.models: delivery.carrier.create() includes unknown fields: use_detailed_pricelist
2018-03-19 17:02:09,356 187 INFO openerp_test openerp.modules.loading: loading delivery_roulier_dpd/data/keychain.xml


@hparfr  Please remove use_detailed_pricelist key

**Comment by bealdav** [comment]
@yvaucher could you have a look here ? Thanks
In production since 1 year, then no problem

**Review by yvaucher — APPROVED** [review]

**Pull request merged**